units/repotracker_activation.go

```diff
@@ -47,6 +47,7 @@ func NewVersionActiationJob(project string, id string) amboy.Job {
 	j.Project = project
 
 	j.SetID(fmt.Sprintf("%s.%s.%s", versionActivationCatchupJobName, project, id))
+	j.SetPriority(-1)
```

Comment:
This maybe should stay at the default, since it's arguably more important than stats jobs.

Reply:
maybe? these jobs will get recreated again and again? arguably they could stay local-only, because they don't have state and are idempotent.

Reply:
True, but we don't want them to block behind stats jobs.